pandoc.cabal

```diff
@@ -599,6 +598,7 @@ library
                    Text.Pandoc.UTF8,
                    Text.Pandoc.Templates,
                    Text.Pandoc.XML,
+                   Text.Pandoc.Network.HTTP,
```

Comment:
I'd prefer to make this an internal (non-exported) module.

Reply:
Done.